static/js/lib/test_utils.js

```diff
@@ -26,3 +29,20 @@ export const findFormikErrorByName = (wrapper: any, name: string) =>
   wrapper
     .find("FormikConnect(ErrorMessageImpl)")
     .filterWhere(node => node.prop("name") === name)
+
+// This is here to support testing components that are wrapped with a
+// context consumer, e.g.:
+//   <SomeContext.Consumer>
+//     {(context) => (<MyComponent myProp={context} />)}
+//   </SomeContext.Consumer>
+// This function takes the wrapped component, props that should be passed into
+// the wrapped component, and the context value for the inner component.
+// It returns the inner component with the context set as desired.
+export const getComponentWithContext = (
```

Comment:
Suggestion: change the return type of this to  `{ inner, outer }`, there might be a case where we want to use the latter down the road and building this is now will save some refactoring in later.